helm/ovn-kubernetes/README.md.gotmpl

```diff
@@ -0,0 +1,98 @@
+{{ template "chart.header" . }}
+-----------------------
+
+{{ template "chart.badgesSection" . }}
+
+{{ template "chart.homepageLine" . }}
+
+{{ template "chart.maintainersSection" . }}
+
+{{ template "chart.sourcesSection" . }}
+
+## Introduction
+
+This helm chart supports deploying OVN K8s CNI in a K8s cluster.
+
+Open Virtual Networking (OVN) Kubernetes CNI is an open source networking and
+network security solution for Kubernetes workloads. It leverages distributed
+OVN SDN control plane and per-node Open vSwitch (OVS) to provide network
+virtualization and network connectivity to K8s Pods. It does so by creating a logical
+network topology using logical constructs such as logical switches (Layer 2) and
+logical routers (Layer 3). The Pod interfaces are represented by logical ports on
+the logical switches. On these logical switch ports, one can specify IP network
+information (IP address and MAC address), anti-spoofing rules (MAC and IP),
+Security Groups, QoS configuration, and so on.
+
+A port, either physical SR-IOV VF or virtual VETH, assigned to a Pod will be associated
+with a corresponding logical port, this will result in applying all the logical port
+configuration on to the physical port. The logical port becomes the API for
+configuring the physical port.
+
+In addition to providing overlay network connectivity for Pods in the K8s cluster,
+OVN K8s CNI supports plethora of advanced networking features, such as
+
+```
+- Optimized and Accelerated K8s Network Policy on Pod's traffic
+- Optimized and Accelerated K8s Service Implementation (aka LoadBalancers and NAT)
+- Optimized and Accelerated Policy Based Routing
+- Multi-Home Pods with an option for Secondary networks to be on a Layer-2
+  Overlay (flat network), Layer-2 Underlay (VLAN-based) on private or public
+  subnets.
+- Optimized and Accelerated K8s Network Policy on Pod's secondary networks
+```
+
+Most of these services are distributed and implemented via a pipeline (series
+of OpenFlow tables with OpenFlow flows) on local OVS switches. These OVS
+pipelines are very amenable to offloading to NIC hardware which should result
+in the best possible networking performance and CPU savings on the host.
+
+OVN K8s CNI architecture is a layered architecture with OVS at the bottom,
+followed by OVN, and finally OVN K8s CNI at the top. Each layer has several
+K8s components - deployments, daemonsets, and statefulsets. Each component at
+every layer is a subchart by itself. Based on the deployment needs, all or
+some of these subcharts are installed to provide the aforementioned OVN K8s
+CNI features, this can be done by editing `tags` section in values.yaml file.
+
+## Quickstart:
```

Comment:
I think we should go a little deeper in the steps described here. I'm currently trying to follow them and am not able to start a very simple cluster with ovnk8. Let's get together and hash it out a bit, please.
Ref: https://gist.github.com/flavio-fernandes/1e21e857d83d4294f8678d5e97eb017d